neural_lam/utils.py

```diff
@@ -108,6 +126,13 @@ def __iter__(self):
         return (self[i] for i in range(len(self)))
 
 
+def zero_index_edge_index(edge_index):
+    """
+    Make both sender and receiver indices of edge_index start at 0
+    """
+    return edge_index - edge_index.min(dim=1, keepdim=True)[0]
+
+
```

Comment:
Now this makes sure that all edge indices are re-indexed to start at 0 immediately when they are loaded, instead of in the InteractionNetworks. This is a good idea, and gets rid of all notions of global node indices throughout the code. Such indices were previously never used, but could be accessed through the loaded edge_index tensors.